app/domain/rewriting_pipeline_execution_dto.py

```
@@ -0,0 +1,11 @@
+from typing import List
```

Ruff (0.8.2)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)